src/app/modules/shared/components/details-fields/details-fields.component.spec.ts

```diff
@@ -468,6 +468,31 @@ describe('DetailsFieldsComponent', () => {
     expect(endDateInput.value).not.toEqual(startDateInput.value);
     expect(startDateInput.value).toEqual(expectedStartDate);
   });
+
+  it('on get actual date should return the current date', () => {
```

Comment:
We should update the references and the name of the test. Something like:
`On get current date should return expected date`